### PR TITLE
Jordan/simple multi denom

### PR DESCRIPTION
**Pull request by jbibla**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmPage.vue [hunk 1]
```diff
@@ -96,6 +96,10 @@ export default {
     signInRequired: {
       type: Boolean,
       default: false
+    },
+    darkBackground: {
```
Comment (faboweb) [inline src/components/common/TmPage.vue]: why? shouldn't the individual section be adjusted?

**Comment by faboweb** [comment]
currency images are too big. validator images are 40px not 3rem
the balance li looks too high imo
icons in the buttons are not centered
data (rewards and available) is missing now

**Comment by jbibla** [comment]
> currency images are too big. validator images are 40px not 3rem

charlies design is 54px. if we need to change the li validator component we can do that separately i think. or would you prefer to make the icons match the current li validator? 

> the balance li looks too high imo

what do you mean, "too high"?

> icons in the buttons are not centered

I'll have a look.

> data (rewards and available) is missing now

yep I see that - trying to debug.

File: src/components/common/TmBalance.vue [hunk 2]
```diff
@@ -172,10 +172,9 @@
           </div>
```
Comment (faboweb) [inline src/components/common/TmBalance.vue]: @Bitcoinera look here for an implementation of our learning from the last 1:1

Comment (Bitcoinera) [reply src/components/common/TmBalance.vue]: ok! I will!

Comment (Bitcoinera) [reply src/components/common/TmBalance.vue]: It is already done, master 🙏

**Comment by Bitcoinera** [comment]
Wow, I love the new Portfolio! I really do ❤️ 

Cannot believe I didn't like it in the beginning. It is so much greater!



BTW, what is SDT exactly?

File: src/components/common/TmBalance.vue (hunk 2 above)
Comment (faboweb) [reply src/components/common/TmBalance.vue]: :P

**Comment by faboweb** [comment]
singapoor dollar?

File: src/components/common/TmBalance.vue (hunk 2 above)
Comment (faboweb) [reply src/components/common/TmBalance.vue]: lol, it was already done. just wanted you to checkout my implementation

**Comment by jbibla** [comment]
https://www.imf.org/en/About/Factsheets/Sheets/2016/08/01/14/51/Special-Drawing-Right-SDR

**Comment by jbibla** [comment]
@faboweb fixed all your comments.

**Comment by faboweb** [comment]
thanks for the SDR clarification

Merged. Looks nice. I think there is a lot of things to improve but this is a solid ground to move forward. Great work.

**Pull request merged**